esp-box/esp-box.c

```diff
@@ -209,7 +209,7 @@ static void lvgl_port_display_init(void)
     esp_lcd_panel_reset(panel_handle);
     esp_lcd_panel_init(panel_handle);
     esp_lcd_panel_mirror(panel_handle, true, true);
-    esp_lcd_panel_disp_on_off(panel_handle, true);
+    esp_lcd_panel_disp_off(panel_handle, false); // In IDF v5.0 display is not turned on automatically
```

Comment:
I would like see here more explaining comment e.g.:
/*
In IDF v5.0 should be called esp_lcd_panel_disp_on_off function for turn on display (display is not turned on automatically in IDF v5.0), but there is used deprecated function esp_lcd_panel_disp_off with the same behavior for compatibility with IDF v4.4
*/

Or something like that.

Reply:
Thanks, comment updated. All commits squashed.